shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -190,6 +190,16 @@ - (void)setupNotificationCenterObservers {
                object:nil];
 
   [center addObserver:self
+             selector:@selector(applicationDidEnterBackground:)
+                 name:UIApplicationDidEnterBackgroundNotification
+               object:nil];
+
+  [center addObserver:self
```

Comment:
It might be a good idea to start using [`-[NSNotificationCenter addObserverForName:object:queue:usingBlock:]`](https://developer.apple.com/reference/foundation/nsnotificationcenter/1411723-addobserverforname?language=objc) so that all the message related to one transition can be read in one single block of code. Just a thought. This way is fine too.